client/src/Components/NewVideo.js

```diff
@@ -0,0 +1,60 @@
+import React, { useState } from "react";
+
+function NewVideo(props) {
+  // const { newVideos, handleNewVideo, handleSubmit} = props
+
+  const [title, setTitle] = useState("");
+  const [url, setUrl] = useState("");
+  const [newVideo, SetNewVideo] = useState({ title: "", url: "" });
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    if (title && url) {
+      // console.log("title and url", title, url);
+      let updatedValue = {
+        title: title,
+        url: url,
+        id: Date.now(),
+        rating: 0,
+      };
+      // console.log("updated value", updatedValue);
+      props.onAddNewVideo(updatedValue)
+      setTitle("")
+      setUrl("")
+    }
+  };
+
+  return (
+    <div className="new-video">
+      <p>Add new video to your list</p>
```

Comment:
Using an `<h3>` element is also a good choice for the title or heading of a section.